storage/pebble/db.go

```diff
@@ -0,0 +1,19 @@
+package pebble
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/pebble"
+	"github.com/onflow/flow-go/storage/pebble/registers"
+)
+
+func InitPebbleDB(dir string) (*pebble.DB, error) {
+	cache := pebble.NewCache(1 << 20)
+	opts := DefaultPebbleOptions(cache, registers.NewMVCCComparer())
+	db, err := pebble.Open(dir, opts)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open db: %w", err)
+	}
+
+	return db, nil
```

Comment:
we need to also return the `cache` so we wan perform a `cache.Unref()` along with closing the db
```suggestion
func InitPebbleDB(dir string) (*pebble.DB, *pebble.Cache, error) {
	cache := pebble.NewCache(1 << 20)
	opts := DefaultPebbleOptions(cache, registers.NewMVCCComparer())
	db, err := pebble.Open(dir, opts)
	if err != nil {
		return nil, fmt.Errorf("failed to open db: %w", err)
	}

	return db, cache nil
```

Reply:
Is Unref() to be called after close or after creation? The documentation says after creation.

>// NewCache creates a new cache of the specified size. Memory for the cache is
// allocated on demand, not during initialization. The cache is created with a
// reference count of 1. Each DB it is associated with adds a reference, so the
// creator of the cache should usually release their reference after the DB is
// created.
//
//	c := pebble.NewCache(...)
//	defer c.Unref()
//	d, err := pebble.Open(pebble.Options{Cache: c})
func NewCache(size int64) *cache.Cache {
	return cache.New(size)
}